src/icons/storybook.ts

```diff
@@ -5,8 +5,10 @@
 import { css, html, LitElement, svg } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import shadowRootMode from '../library/shadow-root-mode.js';
+import final from '../library/final.js';
 
 @customElement('glide-core-example-icon')
+@final
```

Comment:
What's better? Adding an exclusion to `eslint.config.js` or this?

Reply:
Waffling back and forth.

But not modifying `eslint.config.js` for a single use case here feels like the better choice. If we had multiple places outside of `src/*.ts` needing it, I may feel differently. But this feels like a one-off, and adding `@final` here feels less magical than adding a config rule for a single file to me for some odd reason.

Reply:
That's where my head's at too: we modify `eslint.config.js` when we have a second case.